### ISSUE TITLE
sim_telarray ray-tracing tools

### ISSUE THREAD

**Issue by orelgueta** [gammasim/gammasim-tools#178]
Do we have the functionality Konrad describes in [this note](https://forge.in2p3.fr/issues/37526#note-26) in gammasim-tools? That is, calculating the incidence angles on the focal plane (or any of the mirrors)?

**Comment by GernotMaier** [comment]
We don't have this functionality in simtools and we should decide if this is an application needed (and writeup a validation workflow for it).

**Comment by orelgueta** [comment]
Yes, this is necessary functionality. It requires also a special compilation feature to run correctly. That is something we did not consider how to deal with yet, so this is a bigger issue than just writing the application.
